### PR TITLE
First review of Bronson

### PR DESCRIPTION
**Pull request by hershmire**
Not ready to merge just yet. Just setting up this pull request so I can add comments to any changes I make or are planning to make and areas I'm looking for feedback.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,7 +27,7 @@ <h2 class='fontface'>
 </h3>
 <p>
     Bronson is an opinionated module application framework used for building large scalable web applications. Bronson sits on top of RequireJS and JQuery offering
-    libraries for the loading, unloading, messaging and bootstrapping of modules. The idea being that it's impossible to build large manageable JavaScript applications and you are better served in building smaller modular components that when combined form your larger web application.
+    libraries for the loading, unloading, messaging and bootstrapping of modules. The general philosophy of Bronson is to build small modular components that can communicate between each other and when combined, forms your larger web application.
```
Comment (hershmire) [inline index.html:30]: The original wording of the last sentence was tough to comprehend so I took a stab at re-wording it a bit.

**Comment by hershmire** [comment]
What do you think about setting the demo default location to San Francisco rather than Tokyo for better initial readability and understanding?

**Comment by eclifford** [comment]
@hershmire I agree let's move it to SF

**Comment by hershmire** [comment]
@eclifford – This is ready to get merged. Take a look and let me know if everything looks cool and I'll merge it in later.

**Pull request merged**